src/asserts/email-assert.js

```diff
@@ -37,6 +37,8 @@ module.exports = function emailAssert() {
     }
 
     try {
+      console.log('fooo');
```

Comment:
Unintended?

Reply:
Yes, the email assert was a tough nut to crack, since the `extends` does not overwrite the original email assert :laughing:

Reply:
On that note: https://github.com/guillaumepotier/validator.js/pull/67 😄

Reply:
Should there be a [new release](https://github.com/guillaumepotier/validator.js/compare/2.0.3...master) in validator.js?

Reply:
Yeah, not released yet :cry: